jest.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+    clearMocks: true,
+    moduleFileExtensions: ['js', 'ts'],
+    testEnvironment: 'node',
+    testMatch: ['**/*.test.ts'],
+    transform: {
+      '^.+\\.ts$': 'ts-jest'
+    },
+    verbose: true
+  }
```

Comment:
Indentation.

Please update your VS Code settings to insert a new line at the end of all files: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files . This is a common practice especially when Git or other SCM is used to manage the source code.

Reply:
Sure, I will follow this step